OneCommunityGlobal/HGNRest
# Diego add back jest eslint prettier

# Description
Bringing back the configuration updates for jest, eslint, husky and prettier to development. I will be adding the unit and integration tests that have been merged in the past and that are present in this document: https://docs.google.com/spreadsheets/d/1t26Y9h3yRJAfvaWbveRzOz28plXWNgbe3A8Ceiqwyu4/edit#gid=19632796

## Related PRS:
N/A

## Main changes explained:
- Bringing back jest, linting, and prettier to the source code.

## How to test:
1. check into current branch
2. do `npm install`
3. if running on mac do `npm run prepare-macos-linux`
4. create a local test branch and perform any updates there
5. linting, prettier and jest should run when code is committed (jest should run if any test files are added in the staged area)

## Screenshots or videos of changes:

## Note:
Forgive for redoing this. Jae and I needed to merge development features into production solve a deployment issue that was breaking beta stage. Now that it's working, I am bringing back all changes and merged PRs from the past.
